torch/__init__.py

```diff
@@ -1987,6 +1987,7 @@ def _assert(condition, message):
     utils as utils,
     xpu as xpu,
 )
+import torch.optim._multi_tensor  # usort: skip
```

Comment:
May need to add a comment here:

```suggestion
# needs to be before import torch.optim as optim to avoid circular dependencies
import torch.optim._multi_tensor  # usort: skip
```